datahub-frontend/app/controllers/Application.java

```diff
@@ -63,10 +64,16 @@ public Application(Environment environment, @Nonnull Config config) {
    */
   @Nonnull
   private Result serveAsset(@Nullable String path) {
-    InputStream indexHtml = _environment.resourceAsStream("public/index.html");
-    return ok(indexHtml)
-            .withHeader("Cache-Control", "no-cache")
-            .as("text/html");
+    try {
+      InputStream indexHtml = _environment.resourceAsStream("public/index.html");
+      return ok(indexHtml)
+              .withHeader("Cache-Control", "no-cache")
+              .as("text/html");
+    } catch (Exception e) {
```

Comment:
Suggestion: Log the exception here so we have some traceability on what requests failed and the reason why.